cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bootc"
-version = "0.1.0"
+version = "0.1.9"
```

Comment:
Yeah we definitely need to do this and make it part of our release process.  `cargo xtask package` today runs off git tags, but I guess we should verify the two are in sync.